convert2rhel/subscription.py

```diff
@@ -902,6 +906,30 @@ def update_rhsm_custom_facts():
         loggerinst.info("Skipping updating RHSM custom facts.")
 
 
+def get_rhsm_facts():
+    """
+    Open RHSM facts file and parse the facts
+
+    :returns dict: The RHSM facts.
+    """
+    rhsm_facts = {}
+    if tool_opts.no_rhsm:
+        loggerinst.info("Ignoring RHSM facts collection. --no-rhsm is used.")
+        return rhsm_facts
+
+    loggerinst.info("Reading RHSM facts file.")
+    try:
+        with open(RHSM_FACTS_FILE, mode="r") as handler:
+            rhsm_facts = json.load(handler)
+            loggerinst.info("RHSM facts loaded.")
+    except (FileNotFoundError, json.JSONDecodeError) as e:
+        loggerinst.critical_no_exit(
+            "Failed to get the RHSM facts : %s." % e,
+        )
+     finally:
+        return rhsm_facts
```

Comment:
There is an extra space on finally here, but also we don't need `finally` AFAIK
```suggestion
    loggerinst.info("Reading RHSM facts file.")
    try:
        with open(RHSM_FACTS_FILE, mode="r") as handler:
            rhsm_facts = json.load(handler)
            loggerinst.info("RHSM facts loaded.")
    except (FileNotFoundError, json.JSONDecodeError) as e:
        loggerinst.critical_no_exit(
            "Failed to get the RHSM facts : %s." % e,
        )
    return rhsm_facts
```